jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ServletContextRequest.java

```diff
@@ -338,8 +338,8 @@ public Charset getQueryEncoding()
     @Override
     public Object getAttribute(String name)
     {
-        if (AbstractSessionManager.RequestedSession.isApplicableAttribute(name))
-            return _requestedSession.getAttribute(name);
+        if (AbstractSessionManager.RequestedSession.class.getName().equals(name))
```

Comment:
I think that the reverted version is the correct one.

Reply:
@gregw looking at the checkin, it looks like `ee10` and `ee11` versions of this class are deliberately different - do you know why?